doc/installation_instructions/platform/pml-scylla.rst

```diff
@@ -0,0 +1,75 @@
+PML Scylla
+==========
+
+Notes
+-----
+
+As of September 2024, the software stack on Scylla is still being
+finalized. All these instructions should be considered preliminary
+for the time being.
+
+One-time Setup
+--------------
+
+To install SmartSim on Scylla, follow these steps:
+
+**Step 1:** Create and activate a Python virtual environment for SmartSim:
+
+.. code:: bash
+
+    module use module use /scyllapfs/hpe/ashao/smartsim_dependencies/modulefiles
+    module load cudatoolkit cudnn git
+    python -m venv /scyllafps/scratch/$USER/venvs/smartsim
+    source /scyllafps/scratch/$USER/venvs/smartsim/bin/activate
+
+**Step 2:** Build the SmartRedis C++ and Fortran libraries:
+
+.. code:: bash
+
+    git clone https://github.com/CrayLabs/SmartRedis.git
+    cd SmartRedis
+    make lib-with-fortran
+    pip install .
+    cd ..
+
+**Step 3:** Install SmartSim in the conda environment:
+
+.. code:: bash
+
+    pip install git+https://github.com/CrayLabs/SmartSim.git
+
+**Step 4:** Build Redis, RedisAI, the backends, and all the Python packages:
+
+.. code:: bash
+
+    export TORCH_CUDA_ARCH_LIST="8.0 8.6 8.9 9.0" # Workaround for a PyTorch problem
```

Comment:
Do you think any more information should be give for the comment `# Workaround for a PyTorch problem` - maybe in the form of a note -> https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes

Im wondering if user will think itll affect their work? Mayhaps not

Reply:
This is a very preliminary thing for now and it's likely that things will change. The workshop I'm giving tomorrow is basically a "sneak peak" kinda thing. So it should be ok for now. That being said, I agree that I should capture why we have to do this in a note

Reply:
I see!